### PR TITLE
.Tap() -> .FastTap()

### PR DESCRIPTION
**Pull request by carldebilly**
# Feature
UI tests: .FastTap() instead of .Tap()

## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] Tested code with current [supported SDKs](https://github.com/unoplatform/uno/blob/master/README.md#uno-features)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

### CODE REVIEW

File: src/SamplesApp/SamplesApp.UITests/QueryExtensions.cs [hunk 1]
```diff
@@ -49,6 +49,38 @@ public static IAppRect GetRect(this IApp app, string elementName)
...
+		public static void FastTap(this IApp app, string elementName)
+		{
+			var tapPosition = app.GetRect(elementName);
+			app.TapCoordinates(tapPosition.CenterX, tapPosition.Y);
+		}
+
+		public static void FastTap(this IApp app, QueryEx query)
+		{
+			var tapPosition = app.GetRect(query);
+			app.TapCoordinates(tapPosition.CenterX, tapPosition.Y);
```
Comment (jeromelaban) [inline src/SamplesApp/SamplesApp.UITests/QueryExtensions.cs]: CenterY ?

Comment (carldebilly) [reply src/SamplesApp/SamplesApp.UITests/QueryExtensions.cs]: Good catch.

Comment (carldebilly) [reply src/SamplesApp/SamplesApp.UITests/QueryExtensions.cs]: Fixed!

File: src/SamplesApp/SamplesApp.UITests/QueryExtensions.cs [hunk 2]
```diff
@@ -49,6 +49,38 @@ public static IAppRect GetRect(this IApp app, string elementName)
...
+		public static void FastTap(this IApp app, QueryEx query)
+		{
+			var tapPosition = app.GetRect(query);
+			app.TapCoordinates(tapPosition.CenterX, tapPosition.CenterY);
+		}
+
+		public static void FastTap(this IApp app, Func<IAppQuery, IAppQuery> query)
+		{
+			var tapPosition = app.GetRect(query);
+			app.TapCoordinates(tapPosition.CenterX, tapPosition.CenterY);
```
Comment (carldebilly) [inline src/SamplesApp/SamplesApp.UITests/QueryExtensions.cs]: Codacy Issue found: [Refactor this method to add validation of parameter 'app' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4926142)

File: src/SamplesApp/SamplesApp.UITests/QueryExtensions.cs [hunk 3]
```diff
@@ -49,6 +49,38 @@ public static IAppRect GetRect(this IApp app, string elementName)
 		{
 			return app.WaitForElement(elementName).Single().Rect;
 		}
+		public static IAppRect GetRect(this IApp app, QueryEx query)
+		{
+			return app.WaitForElement(query).Single().Rect;
+		}
+		public static IAppRect GetRect(this IApp app, Func<IAppQuery, IAppQuery> query)
+		{
+			return app.WaitForElement(query).Single().Rect;
```
Comment (carldebilly) [inline src/SamplesApp/SamplesApp.UITests/QueryExtensions.cs]: Codacy Issue found: [Refactor this method to add validation of parameter 'app' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4926142)

File: src/SamplesApp/SamplesApp.UITests/QueryExtensions.cs (hunk 2 above)
Comment (carldebilly) [inline src/SamplesApp/SamplesApp.UITests/QueryExtensions.cs]: Codacy Issue found: [Refactor this method to add validation of parameter 'app' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4926142)

File: src/SamplesApp/SamplesApp.UITests/QueryExtensions.cs (hunk 3 above)
Comment (carldebilly) [inline src/SamplesApp/SamplesApp.UITests/QueryExtensions.cs]: Codacy Issue found: [Refactor this method to add validation of parameter 'app' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4926142)

**Pull request merged**